index.js

```diff
@@ -39,6 +41,10 @@ class MultiBuild {
     // The names of the targets.
     this._targets = options.targets;
 
+    // Targets that should not use rollup's cache.
+    this._skipCacheMap = {};
```

Comment:
nit: Let's use a `Set` here rather than a map, that gives you constant-time lookups with simpler initialization